ExampleCheckout/UITests/PaymentSession/Model/Style.swift

```diff
@@ -11,5 +11,5 @@ struct Style: Codable {
     /// Preferred language for payment page. If undefined will be decided upon country information from transaction object.
     ///
     /// Format `<language code>[_<COUNTRY CODE>]`, where `<language code>` is a mandatory part that comply with [ISO 639-1 (alpha-2)](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes), and `<COUNTRY CODE>` is an optional part that comply with [ISO 3166-1 (alpha-2)](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2).
-    var language: String?
+    let language: String
```

Comment:
Should be marked as optional to keep consistency with [a server model](https://www.optile.io/reference#operation/createPaymentSession).

```suggestion
    let language: String?
```